package.json

```diff
@@ -37,10 +37,11 @@
     "lodash.uniqby": "^4.7.0",
     "onecolor": "^3.0.5",
     "prop-types": "^15.6.2",
-    "react": "^16.7.0",
+    "react": "^16.8.3",
     "react-blockies": "^1.3.0",
     "react-container-dimensions": "^1.3.3",
     "react-dom": "^16.7.0",
```

Comment:
I think react-dom also needs to be upgraded to to `16.8.x` to utilise hook support. I got this error: `https://reactjs.org/warnings/invalid-hook-call-warning.html`.

Reply:
Shoot! You are correct. I'll wait till we merge your PR so no need to update here.